CMakeLists.txt

```diff
@@ -516,6 +516,15 @@ find_package(yaml-cpp 0.6.2 REQUIRED HINTS ${YAML_CPP_ROOT})
 get_filename_component(yaml_cpp_dir "${YAML_CPP_INCLUDE_DIR}" ABSOLUTE)
 get_filename_component(yaml_cpp_dir "${yaml_cpp_dir}" DIRECTORY)
 
+find_package(simdjson REQUIRED HINTS ${SIMDJSON_ROOT})
```

Comment:
At least on buster simdjson doesn't provide *version.cmake file, so setting a version for `find_package` is not working.

Reply:
That's alright, we have the same issue for many other dependencies as well.